js/script.js

```diff
@@ -0,0 +1,44 @@
+//
+// Menu mobile toggle active
+//
+
+const h = document.querySelector('header');
+const m = document.querySelector('.menu-bars');
+const i = document.querySelector('.menu-bars i');
+
+// Classes for close menu
+const x = ['fa-bars', 'nav-link'];
+
+m.addEventListener('click', () => {
+  // console.log('hamburger');
```

Comment:
I kindly recommend you not to leave commented codes, your code will look cleaner and more readable

Reply:
Thank you for the advice. I removed the commented code left.